src/style.css

```diff
@@ -1,3 +1,65 @@
+html {
+  --hover_color: #00acdc;
+}
+
 body {
   margin: 0;
+  background-color: gray;
+}
+
+.header {
+  font-family: "Roboto", sans-serif;
+  display: flex;
+  flex-direction: row;
+  background-color: white;
+  width: 100%;
+  height: 60px;
+  justify-content: space-between;
+}
+
+.logo {
+  margin-left: 50px;
+  padding: 10px 0;
+}
+
+.ul {
+  display: flex;
+  align-items: baseline;
+  padding: 0;
+  margin: 0;
+  list-style-type: none;
+  margin-right: 30px;
```

Comment:
margin-right: 30px looks redundant, just reset default browser margin and padding for `<ul> `tag